src/components/MoneyTransactionCreate.js

```diff
@@ -3,12 +3,44 @@ import { Button } from './Button'
 import { SelectInput } from './SelectInput'
 import { TextInput } from './TextInput'
 import styles from './MoneyTransactionCreate.module.css'
+import { useFormik } from 'formik'
+import { object, number } from 'yup'
+
+export const MoneyTransactionCreate = ({ users, creditorId }) => {
+  const transactionSchema = object({
+    amount: number()
+  })
+
+  const formik = useFormik({
+    initialValues: { user: '', amount: '' },
+    validationSchema: transactionSchema,
+    onSubmit: (values) => {
+      // log the values of the form
+      console.log('debitor_id: ' + values.user)
+      console.log('creditor_id: ' + creditorId)
```

Comment:
you could add the creditorId to the form values, then it could be validated there as well, as the value might come from the backend in a future assignment.